CHANGELOG.develop

```diff
@@ -1566,6 +1601,7 @@ Benner and Paweł Siudak):
 - Added support for kaolin themes (thanks to William Roe)
 - Added support for eziam themes (thanks to Benno Fünfstück)
 - Enable italics in spacemacs theme (thanks to Sylvain Benner)
+- Update even more doom themes (thanks to DonHugo69)
```

Comment:
This entry can be combined with the existing entry (just add the author to the "thanks to" of the existing entry).